calendar-bundle/src/Security/Voter/CalendarAccessVoter.php

```diff
@@ -33,6 +36,10 @@ protected function getTable(): string
 
     protected function isGranted(CreateAction|DeleteAction|ReadAction|UpdateAction $action): bool
     {
+        if (!$this->security->isGranted(ContaoCalendarPermissions::USER_CAN_ACCESS_MODULE)) {
```

Comment:
Just out of curiosity, why did you add an extra constant here instead of using `->isGranted(ContaoCorePermissions::USER_CAN_ACCESS_MODULE, 'calendar')`?

Reply:
Just for convenience. Because I can never remember if its calendar, calendars, events etc 😂